_posts/2017/2017-10-29-use-simplecov.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Use simplecov in your Rails test suite'
+description: 'It’s not a silver bullet, but it is useful.'
+layout: article
+category: ruby
+image:
+  base: '2017/use-simplecov'
+  alt: 'Coverage HTML output'
+---
+
+A useful way of assessing the effectiveness of your testing is to use the [`simplecov` gem](https://github.com/colszowka/simplecov) with your Rails application to measure the lines of code that are exercised during your test suite.
```

Comment:
Long sentence. Can you split it into two?

Also `measure the lines of code that are exercised` isn't super clear. You're trying to say 'measure the lines of code that are run/touched', right?